datasheet/Zubax_GNSS2_Datasheet.tex

```diff
@@ -0,0 +1,777 @@
+Allows to view or modify configuration parameters.
+Execute without arguments to get usage info. Supported arguments:
+\begin{itemize}
+\item \textbf{list} - list all parameters, their values and ranges.
+\item \textbf{set <name> <value>} - change parameter value.
+\item \textbf{save} - save all parameters to the non-volatile memory (see the note).
+\item \textbf{erase} - reset all parameters to defaults.
+\end{itemize}
+
+Starting from firmware v4.0, configuration parameters are saved automatically after modification. Thus, it is no longer necessary to execute \textbf{cfg save}.
+
+\textbf{reset}\\
+Restarts the device. Note that sensors will not be restarted.
+
+\textbf{gnssbridge}\\
+Activates the direct bridge connection between USB CLI and the GNSS receiver. This feature allows to receive GNSS information using native u-Blox M8 protocol.
+
+Once the bridge is activated, the state of the device changes as follows until reboot:
+\begin{itemize}
+\item CLI becomes unavailable because it is being used to communicate with the GNSS receiver.
+\item The device stops publishing GNSS messages via UAVCAN.
+\item Status code changes to CRITICAL because GNSS sensor data are not available anymore.
+\end{itemize}
+
+Aside from the above, the device continues to work virtually as usual, e.g., its UAVCAN stack operates normally, other sensors are working (if enabled), etc.
+
+\textbf{help}\\
+Prints the list of available commands
+
+\textbf{bootloader}\\
+Available in firmware v4.0+.\\
+Reboots the board into the bootloader.
+
+\chapter{DroneCode port}
+DroneCode port provides access to JTAG/SWD and UART interfaces. This port allows to update firmware and provides access to the UART interface that is used to log events, report problems, and output measurements in NMEA format. Hardware revisions v2.1 and newer can also be powered via this port.
+
+DroneCode port can be accessed using Zubax DroneCode Probe.
+
+If outputting NMEA over UART is not of interest, this port will not be needed during normal use of the device.
+\section{NMEA output over UART}
+For general information about NMEA output interface, refer to the corresponding section above. Note that NMEA output over UART is disabled by default; refer to the description of configuration parameters to see how to enable it.
+
+The following connector pinout can be used to read NMEA over UART:
+
+\begin{ZubaxSimpleTable}{DroneCode Mini debug connector pinout}{| c | c | c | X |}
+	Pin & Type  & Name                & Comment \\
+	1   & Power & +5VDC               & Power supply (hardware v2.1 and newer). On hardware v2.0 this pin must be left unconnected.\\
+	2 	& OUT	& UART{\_}TX			& NMEA output\\
+	3 & N/C & & \\
+	4 & N/C & & \\
+	5 & N/C & & \\
+	6 & GND & GND & Ground \\
+\end{ZubaxSimpleTable}\\\\\\
+UART parameters are fixed and provided below:
+\begin{itemize}
+\item Baud rate: 115200
+\item Byte size: 8
+\item Parity: None
+\item Stop bits: 1
+\end{itemize}
+
+\chapter{Configuration parameters}
```

Comment:
You're using a lot of manual formatting in this section, such as forced newlines with `\\`, this is not good. Although Latex is not goog at separating the content from the presentation, we should strive to enforce that as much as possible. I suggest you to use either a definition list here, one item per parameter, or you could just crank up a table with one parameter per row.